game-frontend/package.json

```diff
@@ -39,5 +39,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
Please move this block below `dependencies` - so that they're next to eachother.

Reply:
Moved deps above devDeps ✅